app/src/main/java/com/vci/vectorcamapp/imaging/presentation/components/camera/LiveCameraPreviewPage.kt

```diff
@@ -93,12 +95,19 @@ fun LiveCameraPreviewPage(
             modifier = Modifier
                 .fillMaxWidth()
                 .aspectRatio(1f / aspectRatio)
+                .onSizeChanged { outerOverlaySize = it }
```

Comment:
Can you explain how this is working? How is the autofocus part related to displaying the size of the bounding box? I feel like I am understanding the code wrong.